include/mbedtls/sha1.h

```diff
@@ -132,7 +137,9 @@ int mbedtls_sha1_starts_ret( mbedtls_sha1_context *ctx );
  *                 stronger message digests instead.
  *
  * \param ctx      The SHA-1 context.
+ *                 Must not be \c NULL.
  * \param input    The buffer holding the input data.
+ *                 Must not be \c NULL if \p ilen is greater than 0.
```

Comment:
This could additionally express the required size of the buffer, and, optionally, whether it is readable, writable or both.

Reply:
Fixed.